roles/openshift_etcd_migrate/files/validatemigration.py

```diff
@@ -0,0 +1,135 @@
+from subprocess import PIPE, Popen
+import logging
+import json
+import base64
+import optparse
+import six
+
+# logging.basicConfig(level=logging.INFO)
+
+def setOptions():
+	parser = optparse.OptionParser("%prog")
+
+	parser.add_option(
+	    "", "", "--cert-file", dest="certfile", default = "/etc/etcd/peer.crt"
+	)
+
+	parser.add_option(
+	    "", "", "--key-file", dest="keyfile", default = "/etc/etcd/peer.key"
+	)
+
+	parser.add_option(
+	    "", "", "--ca-file", dest="cafile", default = "/etc/etcd/ca.crt"
+	)
+
+	parser.add_option(
+	    "", "", "--etcd-peer", dest="etcdpeer", default = ""
+	)
+
+	parser.add_option(
+	    "", "", "--skip-ttl-keys", dest="skipttlkeys", default = False, action="store_true"
+	)
+
+	return parser.parse_args()
+
+def runCommand(cmd):
+	process = Popen(cmd, stderr=PIPE, stdout=PIPE, shell=True)
+	stdout, stderr = process.communicate()
+	rt = process.returncode
+
+	return stdout.decode('utf-8'), stderr.decode('utf-8'), rt
+
+def keyExists(etcddata):
+	if "count" not in etcddata:
+		return False
+	if etcddata["count"] < 1:
+		return False
+	if "kvs" not in etcddata:
+		return False
+	if len(etcddata["kvs"]) < 1:
+		return False
+	if "value" not in etcddata["kvs"][0]:
+		return False
+	if "key" not in etcddata["kvs"][0]:
+		return False
+
+	return True
+
+# TODO(jchaloup): once done ping kwoodson to test the etcd migration role
+
+if __name__ == "__main__":
+
+	options, args = setOptions()
+	if options.etcdpeer == "":
+		logging.error("--etcd-peer options not set")
+		exit(1)
+
+	ectl2="/usr/bin/etcdctl --cert-file {} --key-file {} --ca-file {} -C https://{}:2379".format(options.certfile, options.keyfile, options.cafile, options.etcdpeer)
```

Comment:
With ``shell=True`` in ``runCommand()`` do you need to give the full path to ``etcdctl``?

@giuseppe will this cause any issues with your etcd container work?